publications/papers/optimal-sizing-paper/main.tex

```diff
@@ -0,0 +1,97 @@
+\documentclass[review]{elsarticle}
```

Comment:
If these files aren't ready, maybe they shouldn't be in the PR?

Reply:
Hmm not sure why these are in this PR. My intention was to remove those files. 
Working on it.